controlpanel/api/models/app.py

```diff
@@ -51,16 +51,23 @@ def customers(self):
     def add_customers(self, emails):
         emails = list(filter(None, emails))
         if emails:
-            auth0.AuthorizationAPI().add_group_members(
-                group_name=self.slug,
-                emails=emails,
-                user_options={"connection": "email"},
-            )
+            try:
+                auth0.AuthorizationAPI().add_group_members(
+                    group_name=self.slug,
+                    emails=emails,
+                    user_options={"connection": "email"},
+                )
+            except auth0.Auth0Error as e:
+                raise AddCustomerError(e)
```

Comment:
we could use `raise AddCustomerError  from e` which enrich the exception with the cause of the exception (as discussed yesterday)